Maple2.File.Parser/Xml/Table/GuildBuff.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Collections.Generic;
+using System.Xml.Serialization;
+using M2dXmlGenerator;
+
+namespace Maple2.File.Parser.Xml.Table;
+
+// ./data/xml/table/guildbuff.xml
+[XmlRoot("ms2")]
+public partial class GuildBuffRoot {
+    [M2dFeatureLocale(Selector = "id|level")] public IList<GuildBuff> _guildBuff;
```

Comment:
Here and elsewhere, `M2dFeatureLocale` fields should be `private` and prefixed with "_". The equivalent public field without the prefix will be generated. You must use the public field to filter the results as the private field includes everything.